### DISCUSSION TITLE
两个点的主id（vid_field）都是字符串类型，不能建立子图吗？

### DISCUSSION THREAD

**Discussion by zjarles** [alibaba/GraphScope#3223] [Q&A]
两个点的id（vid_field）最初是字符串类型的终端号和注册号，建立子图失败。后来将主id都变成了整型，建立子图就成功了。是不支持 字符串类型的id建立子图吗？仅支持整型id建立子图？

**Comment by yecol** [comment]
Hi @zjarles,

Thank you for your interest in GraphScope!
Could you provide more details? e.g., the exact code snippets?
